libs/dal/src/shared/consts/ttl.ts

```diff
@@ -0,0 +1 @@
+export const TTL_EXPIRE_AFTER_AMOUNT = '48h';
```

Comment:
This specific one is only a buffer, I believe what you are referring to is the months addition [here](https://github.com/novuhq/novu/blob/2462dd43e402f19fd2d82cc846f89efe846fc800/libs/dal/src/repositories/base-repository.ts#L89) . And I agree :)
In the future I believe it will be in the database - depending on how we decide to handle user tiers.

Reply:
We are using it in the schema index creation thus my wondering.